android/src/main/cpp/LayoutAnimations.cpp

```diff
@@ -1,4 +1,5 @@
 #include "LayoutAnimations.h"
+#include <vector>
```

Comment:
This is a nitpick but can we move this import above or below? Currently it's between two double-quote import and this is an angle-bracket import.

Reply:
Afaik this should be handled by clang-format.